tests/ted/test.py

```diff
@@ -144,6 +144,12 @@ def run(self):
         return (total, passed, details)
 
     def generate_cases(self):
+        self.platforms = self.config.get('platforms', None)
+        device_name = Path(self.config.get('device', '/dev/dri/renderD128')).name
```

Comment:
I think it will be better if this will be command line option. Also, break change into few: av1 and device path are definitely different things. Do not forget to document new command line option.

Reply:
Done